text_formatter.go

```diff
@@ -72,7 +80,10 @@ func (f *TextFormatter) Format(entry *Entry) ([]byte, error) {
 
 	prefixFieldClashes(entry.Data)
 
-	f.terminalOnce.Do(func() {
+	f.Do(func() {
+		if f.QuoteRune == 0 || !utf8.ValidRune(f.QuoteRune) {
+			f.QuoteRune = '"'
+		}
```

Comment:
Good idea to use this pattern. Since the function is larger now, do you think we could put this in a named function?